### PR TITLE
add how to and all the stuff

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description
- 1st iteration of the README.md see #10 
- I also alphabetically ordered the npm scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,47 @@
 # falcor-postman
 
-falcor-postman
+A tool that enables you to exercise your Express.js app Falcor's endpoint and therefore helps you to validate your queries.
```
Comment (nickbalestra) [inline README.md]: What about something shorter, i really like the graphiQL elevator pitch kind of description, we could borrow from it as we are doing the same thing only for falcor: 
"An graphical interactive in-browser Falcor IDE"

File: README.md [hunk 2]
```diff
@@ -1,13 +1,47 @@
...
+const app = express();
+
+const options = { middlewarePath: '/falcor-postman', falcorPath: '/model.json', app };
+
+app.use(falcorPostman(options));
+```
+
+Where **options** is an object with the following properties:
+
+name|type|description|example
```
Comment (nickbalestra) [inline README.md]: Perhaps instead of "exmple" we could rename could column to "default" ?

File: README.md [hunk 3]
```diff
@@ -1,13 +1,47 @@
...
-falcor-postman
+A tool that enables you to exercise your Express.js app Falcor's endpoint and therefore helps you to validate your queries.
+
+## Usage
+
+Npm install using the following command:
 
 ```
-npm i
+npm i -S falcor-postman
```
Comment (nickbalestra) [inline README.md]: For the sake of documentation, I will go with the full form: 
`npm install --save falcor-postman`

**Review by nickbalestra — COMMENTED** [review]
GJ, over to you @mattiaerre

File: README.md (hunk 2 above)
Comment (mattiaerre) [reply README.md]: much better you are right

File: README.md (hunk 1 above)
Comment (mattiaerre) [reply README.md]: what about: "A graphical interactive in-browser Falcor queries validator IDE"

Comment (nickbalestra) [reply README.md]: great one, me likes it!

**Comment by nickbalestra** [comment]
Thanks @mattiaerre!

**Pull request merged**